style.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: aqua;
+}
```

Comment:
- [ ] The class name that was given to your h1 in the HTML file should be used here to style the content

E.G:
``` CSS
.header {
    color: aqua;
  }
```

With this, you will be sure you follow the project requirement